test-data/unit/check-parameter-specification.test

```diff
@@ -1127,3 +1127,38 @@ def func(callback: Callable[P1, str]) -> Callable[P1, str]:
         return "foo"
     return inner
 [builtins fixtures/paramspec.pyi]
+
+[case testParamSpecTestPropAccess]
+from typing import Callable
+from typing_extensions import ParamSpec
+
+P1 = ParamSpec("P1")
+
+def func1(callback: Callable[P1, str]) -> Callable[P1, str]:
+    def inner(
+        *args: P1.typo,  # E: Use "P1.args" for variadic "*" parameter \
+                         # E: Name "P1.typo" is not defined
```

Comment:
How hard would it be to make this say something like "ParamSpec has no attribute 'typo'" instead?

Reply:
Quite hard :)

Right now `checkmember` is not involved at all, this is `semanal`-level check.
We can change the error message (special case it for `ParamSpec`), but I don't think this is a good idea.